test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -372,7 +372,9 @@ describe('Dropdown', () => {
         .find('DropdownItem')
         .first()
         .simulate('click')
-        .should.have.prop('active', true)
+        // `first().simulate('click')` closes the Dropdown so can't verify `prop('active', true)`
+        // TODO: Find a way to verify `first()` has `prop('active', true)`
+        // .should.have.prop('active', true)
```

Comment:
I would think re-opening the dropdown would retain the active item.  You could simply try something like (untested):

```jsx
wrapper
  .find('DropdownItem')
  .first()
  .simulate('click')

// re-open the dropdown
wrapper.setProps({ open: true })

wrapper
  .find('DropdownItem')
  .first()
  .should.have.prop('active', true)
```